app/serializers/export/comment_serializer.rb

```diff
@@ -0,0 +1,8 @@
+module Export
+  class CommentSerializer < ActiveModel::Serializer
+    attributes :text,
+               :author_id,
```

Comment:
`author_id` and `author_signature` are probably irrelevant for a user, even in a pod migration situation (I think). Could we instead add here author diaspora handle? That is the network wide unique identifier that is actually usable. GUID is of course too but that doesn't say anything to users looking at the export.